ros_awsiot_agent/setup.py

```diff
@@ -1,9 +1,33 @@
-from catkin_pkg.python_setup import generate_distutils_setup
 from setuptools import setup
+import os
+from glob import glob
 
-# fetch values from package.xml
-setup_args = generate_distutils_setup(
-    packages=["ros_awsiot_agent"], package_dir={"": "src"}
-)
+package_name = 'ros_awsiot_agent'
 
-setup(**setup_args)
+setup(
+    name=package_name,
+    version='0.0.1',
+    packages=[package_name],
+    data_files=[
+        ('share/' + package_name, ['package.xml']),
```

Comment:
```suggestion
        ('share/' + package_name, ['package.xml']),
        ('share/ament_index/resource_index/packages', ['resource/' + package_name]),
```

It would be better to place an empty file `ros_awsiot_agent/resource/ros_awsiot_agent`

> $ colcon build --packages-up-to ros_awsiot_agent
> Starting >>> ros_awsiot_agent
> [0.360s] WARNING:colcon.colcon_ros.task.ament_python.build:Package 'ros_awsiot_agent' doesn't explicitly install a marker in the package index (colcon-ros currently does it implicitly but that fallback will be removed in the future)

https://robotics.stackexchange.com/questions/97794/ament-python-package-doesnt-explicitly-install-a-marker-in-the-package-index

Reply:
[162a78f](https://github.com/whill-labs/ros_awsiot/pull/11/commits/162a78ff2117e27f0a300d824bcda277bbe7d349)